spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+describe ReviewsController do
+  describe "POST create" do
+    context "with authenticated user" do
+      let(:video) { Fabricate(:video) }
+      let(:user) { Fabricate(:user) }
+      let(:log_in) { session[:user_id] = Fabricate(:user).id }
```

Comment:
Also, good job sensing the need to refactor this. You will learn another way to handle these refactorings soon.